web/yo/app/scripts/app.js

```diff
@@ -390,6 +390,7 @@ OncoKB.setUp = function(object) {
 OncoKB.keyMappings = {type: {TSG: '', OCG: ''}};
 
 OncoKB.initialize = function() {
+    firebase.initializeApp(OncoKB.config.firebaseConfig);
```

Comment:
Where this firebase var imported? From window object? Could you register it when initializing the app? Similarly to underscore.